ksql-engine/src/main/java/io/confluent/ksql/structured/SchemaKGroupedStream.java

```diff
@@ -38,24 +40,24 @@
 public class SchemaKGroupedStream {
 
   final ExecutionStep<KGroupedStreamHolder> sourceStep;
+  final LogicalSchema schema;
   final KeyFormat keyFormat;
   final KeyField keyField;
-  final List<SchemaKStream> sourceSchemaKStreams;
```

Comment:
these aren't used anymore now that we build the plan summary string from the execution steps